LICENSE-binary

```diff
@@ -320,11 +322,14 @@ com.github.mifmif:generex
 com.github.stephenc.jcip:jcip-annotations
 com.google.code.findbugs:jsr305
 com.google.code.gson:gson
+com.google.flatbuffers:flatbuffers-java
+com.google.guava:guava
```

Comment:
Do we really bring in unshaded Guava?

Reply:
Wait, where are all these additional license entries coming from? according to the deps file, the assembly doesn't bring them in. Are you seeing these in the distro jars/ folder?

Reply:
> Wait, where are all these additional license entries coming from? according to the deps file, the assembly doesn't bring them in. Are you seeing these in the distro jars/ folder?

Yes, it bring guava in dist/jars/

Reply:
OK. BTW do you have the output of `ls jars/` on the distro? just for reference here.

Reply:
```
accessors-smart-1.2.jar
activation-1.1.1.jar
aircompressor-0.10.jar
algebra_2.12-2.0.0-M2.jar
antlr4-runtime-4.7.1.jar
antlr-runtime-3.5.2.jar
aopalliance-1.0.jar
aopalliance-repackaged-2.5.0.jar
arpack_combined_all-0.1.jar
arrow-format-0.12.0.jar
arrow-memory-0.12.0.jar
arrow-vector-0.12.0.jar
audience-annotations-0.5.0.jar
automaton-1.11-8.jar
avro-1.8.2.jar
avro-ipc-1.8.2.jar
avro-mapred-1.8.2-hadoop2.jar
bonecp-0.8.0.RELEASE.jar
breeze-macros_2.12-1.0.jar
breeze_2.12-1.0.jar
cats-kernel_2.12-2.0.0-M4.jar
chill-java-0.9.3.jar
chill_2.12-0.9.3.jar
commons-beanutils-1.9.3.jar
commons-cli-1.2.jar
commons-codec-1.10.jar
commons-collections-3.2.2.jar
commons-compiler-3.0.15.jar
commons-compress-1.8.1.jar
commons-configuration2-2.1.1.jar
commons-crypto-1.0.0.jar
commons-daemon-1.0.13.jar
commons-dbcp-1.4.jar
commons-httpclient-3.1.jar
commons-io-2.4.jar
commons-lang3-3.8.1.jar
commons-lang-2.6.jar
commons-logging-1.1.3.jar
commons-math3-3.4.1.jar
commons-net-3.1.jar
commons-pool-1.5.4.jar
commons-text-1.6.jar
compress-lzf-1.0.3.jar
core-1.1.2.jar
curator-client-2.13.0.jar
curator-framework-2.13.0.jar
curator-recipes-2.13.0.jar
datanucleus-api-jdo-4.2.4.jar
datanucleus-core-4.1.17.jar
datanucleus-rdbms-4.1.19.jar
derby-10.12.1.1.jar
dnsjava-2.1.7.jar
dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
ehcache-3.3.1.jar
flatbuffers-java-1.9.0.jar
generex-1.0.2.jar
geronimo-jcache_1.0_spec-1.0-alpha-1.jar
gson-2.2.4.jar
guava-14.0.1.jar
guice-4.0.jar
guice-servlet-4.0.jar
hadoop-annotations-3.2.0.jar
hadoop-auth-3.2.0.jar
hadoop-client-3.2.0.jar
hadoop-common-3.2.0.jar
hadoop-hdfs-client-3.2.0.jar
hadoop-mapreduce-client-common-3.2.0.jar
hadoop-mapreduce-client-core-3.2.0.jar
hadoop-mapreduce-client-jobclient-3.2.0.jar
hadoop-yarn-api-3.2.0.jar
hadoop-yarn-client-3.2.0.jar
hadoop-yarn-common-3.2.0.jar
hadoop-yarn-registry-3.2.0.jar
hadoop-yarn-server-common-3.2.0.jar
hadoop-yarn-server-web-proxy-3.2.0.jar
HikariCP-2.5.1.jar
hive-beeline-2.3.6.jar
hive-cli-2.3.6.jar
hive-common-2.3.6.jar
hive-exec-2.3.6-core.jar
hive-jdbc-2.3.6.jar
hive-llap-common-2.3.6.jar
hive-metastore-2.3.6.jar
hive-serde-2.3.6.jar
hive-service-rpc-2.3.6.jar
hive-shims-0.23-2.3.6.jar
hive-shims-2.3.6.jar
hive-shims-common-2.3.6.jar
hive-shims-scheduler-2.3.6.jar
hive-storage-api-2.6.0.jar
hive-vector-code-gen-2.3.6.jar
hk2-api-2.5.0.jar
hk2-locator-2.5.0.jar
hk2-utils-2.5.0.jar
hppc-0.7.2.jar
htrace-core4-4.1.0-incubating.jar
httpclient-4.5.6.jar
httpcore-4.4.10.jar
istack-commons-runtime-3.0.8.jar
ivy-2.4.0.jar
jackson-annotations-2.9.10.jar
jackson-core-2.9.10.jar
jackson-core-asl-1.9.13.jar
jackson-databind-2.9.10.jar
jackson-dataformat-yaml-2.9.10.jar
jackson-jaxrs-base-2.9.5.jar
jackson-jaxrs-json-provider-2.9.5.jar
jackson-mapper-asl-1.9.13.jar
jackson-module-jaxb-annotations-2.9.10.jar
jackson-module-paranamer-2.9.10.jar
jackson-module-scala_2.12-2.9.10.jar
jakarta.annotation-api-1.3.4.jar
jakarta.inject-2.5.0.jar
jakarta.ws.rs-api-2.1.5.jar
jakarta.xml.bind-api-2.3.2.jar
janino-3.0.15.jar
javassist-3.22.0-CR2.jar
javax.el-3.0.1-b11.jar
javax.inject-1.jar
javax.jdo-3.2.0-m3.jar
javax.servlet-api-3.1.0.jar
javolution-5.5.1.jar
jaxb-api-2.2.11.jar
jaxb-runtime-2.3.2.jar
jcip-annotations-1.0-1.jar
jcl-over-slf4j-1.7.16.jar
jdo-api-3.0.1.jar
jersey-client-2.29.jar
jersey-common-2.29.jar
jersey-container-servlet-2.29.jar
jersey-container-servlet-core-2.29.jar
jersey-hk2-2.29.jar
jersey-media-jaxb-2.29.jar
jersey-server-2.29.jar
jetty-webapp-9.4.18.v20190429.jar
jetty-xml-9.4.18.v20190429.jar
JLargeArrays-1.5.jar
jline-2.14.6.jar
joda-time-2.9.3.jar
jodd-core-3.5.2.jar
jpam-1.1.jar
json4s-ast_2.12-3.6.6.jar
json4s-core_2.12-3.6.6.jar
json4s-jackson_2.12-3.6.6.jar
json4s-scalap_2.12-3.6.6.jar
json-1.8.jar
json-smart-2.3.jar
jsp-api-2.1.jar
jsr305-3.0.0.jar
jta-1.1.jar
JTransforms-3.1.jar
jul-to-slf4j-1.7.16.jar
kerb-admin-1.0.1.jar
kerb-client-1.0.1.jar
kerb-common-1.0.1.jar
kerb-core-1.0.1.jar
kerb-crypto-1.0.1.jar
kerb-identity-1.0.1.jar
kerb-server-1.0.1.jar
kerb-simplekdc-1.0.1.jar
kerb-util-1.0.1.jar
kerby-asn1-1.0.1.jar
kerby-config-1.0.1.jar
kerby-pkix-1.0.1.jar
kerby-util-1.0.1.jar
kerby-xdr-1.0.1.jar
kryo-shaded-4.0.2.jar
kubernetes-client-4.4.2.jar
kubernetes-model-4.4.2.jar
kubernetes-model-common-4.4.2.jar
leveldbjni-all-1.8.jar
libfb303-0.9.3.jar
libthrift-0.12.0.jar
log4j-1.2.17.jar
logging-interceptor-3.12.0.jar
lz4-java-1.6.0.jar
machinist_2.12-0.6.8.jar
macro-compat_2.12-1.1.1.jar
mesos-1.4.0-shaded-protobuf.jar
metrics-core-3.1.5.jar
metrics-graphite-3.1.5.jar
metrics-json-3.1.5.jar
metrics-jvm-3.1.5.jar
minlog-1.3.0.jar
mssql-jdbc-6.2.1.jre7.jar
netty-all-4.1.39.Final.jar
nimbus-jose-jwt-4.41.1.jar
objenesis-2.5.1.jar
okapi-shade-0.4.2.jar
okhttp-2.7.5.jar
okhttp-3.8.1.jar
okio-1.13.0.jar
opencsv-2.3.jar
orc-core-1.5.6.jar
orc-mapreduce-1.5.6.jar
orc-shims-1.5.6.jar
oro-2.0.8.jar
osgi-resource-locator-1.0.3.jar
paranamer-2.8.jar
parquet-column-1.10.1.jar
parquet-common-1.10.1.jar
parquet-encoding-1.10.1.jar
parquet-format-2.4.0.jar
parquet-hadoop-1.10.1.jar
parquet-jackson-1.10.1.jar
protobuf-java-2.5.0.jar
py4j-0.10.8.1.jar
pyrolite-4.30.jar
re2j-1.1.jar
RoaringBitmap-0.7.45.jar
scala-collection-compat_2.12-2.1.1.jar
scala-compiler-2.12.10.jar
scala-library-2.12.10.jar
scala-parser-combinators_2.12-1.1.2.jar
scala-reflect-2.12.10.jar
scala-xml_2.12-1.2.0.jar
shapeless_2.12-2.3.3.jar
shims-0.7.45.jar
slf4j-api-1.7.16.jar
slf4j-log4j12-1.7.16.jar
snakeyaml-1.23.jar
snappy-java-1.1.7.3.jar
spark-catalyst_2.12-3.0.0-SNAPSHOT.jar
spark-core_2.12-3.0.0-SNAPSHOT.jar
spark-cypher_2.12-3.0.0-SNAPSHOT.jar
spark-graph-api_2.12-3.0.0-SNAPSHOT.jar
spark-graph_2.12-3.0.0-SNAPSHOT.jar
spark-graphx_2.12-3.0.0-SNAPSHOT.jar
spark-hive-thriftserver_2.12-3.0.0-SNAPSHOT.jar
spark-hive_2.12-3.0.0-SNAPSHOT.jar
spark-kubernetes_2.12-3.0.0-SNAPSHOT.jar
spark-kvstore_2.12-3.0.0-SNAPSHOT.jar
spark-launcher_2.12-3.0.0-SNAPSHOT.jar
spark-mesos_2.12-3.0.0-SNAPSHOT.jar
spark-mllib-local_2.12-3.0.0-SNAPSHOT.jar
spark-mllib_2.12-3.0.0-SNAPSHOT.jar
spark-network-common_2.12-3.0.0-SNAPSHOT.jar
spark-network-shuffle_2.12-3.0.0-SNAPSHOT.jar
spark-repl_2.12-3.0.0-SNAPSHOT.jar
spark-sketch_2.12-3.0.0-SNAPSHOT.jar
spark-sql_2.12-3.0.0-SNAPSHOT.jar
spark-streaming_2.12-3.0.0-SNAPSHOT.jar
spark-tags_2.12-3.0.0-SNAPSHOT.jar
spark-tags_2.12-3.0.0-SNAPSHOT-tests.jar
spark-unsafe_2.12-3.0.0-SNAPSHOT.jar
spark-yarn_2.12-3.0.0-SNAPSHOT.jar
spire-macros_2.12-0.17.0-M1.jar
spire-platform_2.12-0.17.0-M1.jar
spire-util_2.12-0.17.0-M1.jar
spire_2.12-0.17.0-M1.jar
ST4-4.0.4.jar
stax2-api-3.1.4.jar
stax-api-1.0.1.jar
stream-2.9.6.jar
super-csv-2.2.0.jar
token-provider-1.0.1.jar
transaction-api-1.1.jar
univocity-parsers-2.7.3.jar
validation-api-2.0.1.Final.jar
velocity-1.5.jar
woodstox-core-5.0.3.jar
xbean-asm7-shaded-4.14.jar
xz-1.5.jar
zjsonpatch-0.3.0.jar
zookeeper-3.4.14.jar
zstd-jni-1.4.2-1.jar
```